pkg/storaged/format-dialog.jsx

```diff
@@ -439,11 +490,11 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
 
                 function maybe_mount(new_path) {
                     const path = new_path || block.path;
-                    if (is_filesystem(vals) && vals.mount_options.auto)
+                    if (is_filesystem(vals) && mount_now)
                         return (client.wait_for(() => block_fsys_for_block(path))
                                 .then(block_fsys => client.mount_at(client.blocks[block_fsys.path],
                                                                     mount_point)));
-                    if (is_encrypted(vals) && vals.mount_options && !vals.mount_options.auto)
+                    if (is_encrypted(vals) && is_filesystem(vals) && !mount_now)
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221115-130151-a219e579-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)